### PR TITLE
Feat/disable geocoding option

### PR DESCRIPTION
**Pull request by Ryoken**
#### What's this PR do?
Adds a setting to organizations that disables geocoding.
#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/1869

### CODE REVIEW

File: seed/static/seed/js/controllers/mapping_controller.js [hunk 1]
```diff
@@ -502,16 +502,20 @@ angular.module('BE.seed.controller.mapping', [])
...
-          $scope.taxlot_geocoding_columns_array = _.map(geocoding_columns.TaxLotState, function (column_name) {
-            return _.find($scope.mappable_taxlot_columns, {column_name: column_name}).display_name;
-          });
-          $scope.taxlot_geocoding_columns = $scope.taxlot_geocoding_columns_array.join(', ');
+              $scope.taxlot_geocoding_columns_array = _.map(geocoding_columns.TaxLotState, function (column_name) {
+                return _.find($scope.mappable_taxlot_columns, {column_name: column_name}).display_name;
+              });
+              $scope.taxlot_geocoding_columns = $scope.taxlot_geocoding_columns_array.join(', ');
+            });
+          }
```
Comment (adrian-lara) [inline seed/static/seed/js/controllers/mapping_controller.js:518]: Left is activated and right is deactivated. Nice catch to hide the relevant help text! 
Screen Shot 2020-11-11 at 5 29 35 PM

**Review by adrian-lara — COMMENTED** [review]
This looks great so far! One case that wasn't mentioned to you previously: Users can geocode records on the inventory list page.
Screen Shot 2020-11-11 at 5 04 42 PM

Right now, if I have an API key but disable geocoding, I see this and can click "Geocode".
Screen Shot 2020-11-11 at 5 05 01 PM

Maybe it's worth doing the enabled/disabled check and adding to the message we show when there isn't an API key.
Screen Shot 2020-11-11 at 5 38 32 PM


Otherwise this looks good during the import flow! See code comment and the following screenshot showing the results of importing the same file with geocoding enabled (left) and disabled (right):
Screen Shot 2020-11-11 at 5 35 21 PM

File: seed/static/seed/js/controllers/mapping_controller.js (hunk 1 above)
Comment (Ryoken) [reply seed/static/seed/js/controllers/mapping_controller.js:518]: Added a message to the geocode modal if disabled.

**Review by adrian-lara — APPROVED** [review]
Looks good and works as expected! Thanks!

The last requested change:
Screen Shot 2020-11-12 at 4 08 15 PM

**Pull request merged**